drivers/hmis/app/graphql/types/hmis_schema/has_events.rb

```diff
@@ -32,9 +32,8 @@ def resolve_events(scope = object.events, **args)
 
       private
 
-      def apply_event_arguments(scope, user: current_user, sort_order: :date)
-        events_scope = scope.viewable_by(user)
-        events_scope.sort_by_option(sort_order) if sort_order.present?
+      def apply_event_arguments(scope, sort_order: :event_date)
+        scope.sort_by_option(sort_order) if sort_order.present?
```

Comment:
I'm still seeing the same error, now it's just `undefined method `sort_by_option' for :events:Symbol`. Are you able to test this in graphiql @ZachCMP?

Reply:
I had trouble doing that because I have no events in my local DB